blog/content/post/2022-01-03-pcie_screamer.md

```diff
@@ -0,0 +1,218 @@
+---
+title: PCIe Screamer first look
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/PCIExpress.jpg
+author: igor.bagnucki
+layout: post
+published: true
+date: 2022-01-03
+archives: "2022"
+
+tags:
+  - pcie
+  - pciescreamer
+  - screamer
+  - lambda
+  - lambdaconcept
+  - sniffer
+  - sniff
+  - sniffing
+  - pci
+  - fpga
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Why test PCIe sniffing
+
+Information security is a complicated topic. It needs to be considered in
+every step of system design. Additionally, for every system, the threat model
+may be different.
+As a system includes multiple PCIe devices, that, i.e., can use the system's
+memory and be shared to an untrusted virtual machine or can themself be
+malicious there is a need to protect the memory from DMA attacks executed from
+the PCIe bus.
+
+To avoid this risk, there is an option to enable [IOMMU](https://blog.3mdeb.com/2021/2021-01-13-iommu/)
+what should fix the problem, but does it?
+
+To verify that IOMMU protects from DMA attacks, this functionality should
+be tested, or in other words, it should be tried to make such an attack on a
+system.
+
+## What is PCIeScreamer
```

Comment:
In my opinion there is lack of information what version of "PCIScreamer" had been used in experiment. There are two majot versions of this device: version2 and version 2 and version 4. On photos posted on project WWW page one can see that JTAG headers a not soldered.